conf/config.properties.example

```diff
@@ -384,7 +384,8 @@ client.roles.cmpdreg.chemistRole=ROLE_CMPDREG-USERS
erClassName":"ExampleThingBrowserController","autoLaunchName":"example_thing_browser"},{"isHeader":true,"menuName":"Search and Edit","collapsible":false,"requireUserRoles":["${client.roles.acas.userRole}"]},{"isHeader":false,"menuName":"Protocol Browser","mainControllerClassName":"ProtocolBrowserController","requireUserRoles":["${client.roles.acas.userRole}"]},{"isHeader":false,"menuName":"Experiment Browser","mainControllerClassName":"ExperimentBrowserController","requireUserRoles":["${client.roles.acas.userRole}"]},{"isHeader":true,"menuName":"Admin","collapsible":false,"requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"Project Editor","mainControllerClassName":"ProjectController","autoLaunchName":"project","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"Project Browser","mainControllerClassName":"ProjectBrowserController","requireUserRoles":["${client.roles.acas.userRole}"]},{"isHeader":false,"menuName":"Author Editor","mainControllerClassName":"AuthorEditorController","autoLaunchName":"author","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"Author Browser","mainControllerClassName":"AuthorBrowserController","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"System Test","mainControllerClassName":"SystemTestController","autoLaunchName":"system_test","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"Label Sequence","mainControllerClassName":"ACASLabelSequenceController","autoLaunchName":"acasLabelSequence","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"Additional Assay Scientists","mainControllerClassName":"AssayScientistBrowserController","autoLaunchName":"assay_scientist_browser","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":true,"menuName":"CmpdReg Admin","requireUserRoles":["${client.roles.cmpdreg.adminRole}"],"collapsible":false},{"isHeader":false,"menuName":"CmpdReg Salts Browser","mainControllerClassName":"SaltBrowserController","requireUserRoles":["${client.roles.acas.adminRole}"]},{"isHeader":false,"menuName":"CmpdReg Standardization","mainControllerClassName":"StandardizationController","autoLaunchName":"standardization","requireUserRoles":["${client.roles.cmpdreg.adminRole}"]},{"isHeader":false,"menuName":"CmpdReg Vendors","mainControllerClassName":"VendorBrowserController","autoLaunchName":"vendor_browser","requireUserRoles":["${client.roles.cmpdreg.adminRole}"]},{"isHeader":false,"menuName":"CmpdReg Stereo Categories","mainControllerClassName":"StereoCategoryBrowserController","autoLaunchName":"stereo_category_browser","requireUserRoles":["${client.roles.cmpdreg.adminRole}"]},{"isHeader":false,"menuName":"Additional CmpdReg Scientists","mainControllerClassName":"CompoundScientistBrowserController","autoLaunchName":"compound_scientist_browser","requireUserRoles":["${client.roles.cmpdreg.adminRole}"]}]
```

Comment:
nit: Other modules are called `CmpdReg Vendors`, `CmpdReg Stereo Categories` but yours is called `CmpdReg Salts Browser`. You should shorten it to `CmpdReg Salts`